.github/workflows/build-docs.yml

```diff
@@ -21,16 +21,17 @@ jobs:
         run: |
           # install pip=>20.1 to use "pip cache dir"
           python3 -m pip install --upgrade pip
-      - name: Get pip cache dir
-        id: pip-cache
-        run: echo "::set-output name=dir::$(pip cache dir)"
+      - name: Set Variables
+        id: set_variables
+        shell: bash
+        run: |
```

Comment:
So can you help me understand what this run step is doing @yihong0618 no critique here i'm just trying to understand the fix! i see that we need to implement the fix from that blog you linked to!!

Reply:
![image](https://user-images.githubusercontent.com/15976103/203904655-b939f110-fd26-40a6-8a52-97d861d47cea.png)

different python version + executable generate different hashid as key

Reply:
thanks!